FEWD/Week03/lesson06/homework/startup_matchmaker/developers.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Developers</title>
+
+  <link rel="stylesheet" href="./style/reset.css">
+  <link rel="stylesheet" href="./style/style.css">
+
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,600,700,400,400italic" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Oswald:400,300,700" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400italic,700,300" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+  <header>
+    <h1>Startup Matchmaker</h1>
```

Comment:
It's pretty standard to make the logo a link to the homepage, you might want to add that in here